grpc/proto/search_get_v1.proto

```diff
@@ -0,0 +1,267 @@
+syntax = "proto3";
+
+package weaviategrpc;
+
+import "base.proto";
+import "google/protobuf/struct.proto";
+
+option go_package = "github.com/weaviate/weaviate/grpc/generated;protocol";
+option java_package = "io.weaviate.grpc.protocol";
+option java_outer_classname = "WeaviateProtoSearchGet";
+
+message SearchRequestV1 {
+  //required
+  string collection = 1;
+
+  // parameters
+  string tenant = 10;
```

Comment:
I think that tenant can be marked as `optional` here

Reply:
as written in the comment above, inside if weaviate we use the default values (0 for numbers, "" for strings) as a marker for not enabled. So if we would make after optional, we would need to do:
```
if request.tenant != nil && {
    out.tenant = *(request.tenant)
} else{
    out.tenant = ""
}
```

instead of just doing 
```
out.Tenant = req.Tenant
```

Reply:
ok, got it